README.md

```diff
@@ -38,13 +38,13 @@ You could also use the console built in to VSCode (or your preferred IDE/editor)
 
 We'll be using ESLint to check our code syntax. Wikipedia defines linter or lint thus: _tools that analyse source code to flag programming errors, bugs, stylistic errors, and suspicious constructs_. ESLint will help you to write better code.
 
-VSCode has an [ESLint extension](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint), as do most code editors.
+VSCode has an [ESLint extension](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint), as do most code editors. Go ahead and install this now.
```

Comment:
Installation is optional, so this addition is incorrect. I am not insisting that the reader install VSCode or any particular extensions. That said, I'll adapt these as it's good to remind them that now would be a good time to install same.